<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>726</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>View LDR</string>
  </property>
  <widget class="QTableWidget" name="tableWidgetLDR">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>691</width>
     <height>391</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Filename = </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="textbox_FilenameLDR">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>631</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
